generators/ruby/faraday/templates/README.mustache

```diff
@@ -9,32 +9,37 @@ Documentation can be found at https://documentation.onfido.com
 
 This version uses Onfido API v3.6 and is compatible with Ruby 2.4 onwards. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
 
-## Installation
+## Installation & Usage
 
-Add this line to your application's Gemfile:
+### Requirements
+
+Ruby {{{generatorLanguageVersion}}}
+
+### Installation
 
 ```ruby
-gem 'onfido', '~> 2.0.1'
+gem {{{gemName}}}, '~> 2.0.1'
 ```
 
-## Getting started
-
 Configure with your API token and region:
 
 ```ruby
-onfido = Onfido::API.new(
-  api_key: ENV['ONFIDO_API_KEY'],
-  # Supports :eu, :us and :ca. Previously defaulted to :eu.
-  region: :eu
-)
+require {{{gemName}}}
+
+Onfido.configure do |config|
+  config.api_token = ENV["ONFIDO_API_TOKEN"]
+  config.region = config.region[:EU]
+end
```

Comment:
Would be nice to provide also an example how to change the default timeout as in the other languages. 

Btw something we've probably missed is setting a default timeouts to 30s (as in other languages, default value shoud be 0 - infinite).